modules/littlefs/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+if(CONFIG_FILE_SYSTEM_LITTLEFS)
+  add_library(LITTLEFS INTERFACE)
+
+  target_include_directories(LITTLEFS INTERFACE ${ZEPHYR_LITTLEFS_MODULE_DIR})
+
+  zephyr_library()
+  zephyr_library_sources(
+      ${ZEPHYR_LITTLEFS_MODULE_DIR}/lfs.c
+      ${ZEPHYR_LITTLEFS_MODULE_DIR}/lfs_util.c
+      zephyr_lfs_crc.c
+      )
```

Comment:
Closing bracket is indented differently than in line 18

Reply:
Fixed.

Danger of simply moving code unaltered from one repo to another and then make extra additions :stuck_out_tongue: